tools/local_fix_format.sh

```diff
@@ -29,6 +29,8 @@ else
   verbose=0
 fi
 
+. ci/build_setup.sh
```

Comment:
```suggestion
if [[ "$BUILD_REASON" == "local-ci" ]]'; then
    . ci/build_setup.sh
fi
```

the script can then be run inside the Docker container with

```console

BUILD_REASON=local-ci ./ci/run_envoy_docker.sh ./tools/local_fix_fix_format.sh
```

the reason for suggesting this is that the script is already used afaiaa _outside_ of the container in a local env that does not require/want this

the reason for suggesting `BUILD_REASON` specifically is that its an azp var that is currently passed through the Docker container, but is not azp-specific and is safe to overload in this way

Reply:
thanks, done